packages/agreement-process/src/model/domain/models.ts

```diff
@@ -48,3 +52,52 @@ export const CompactEService = z.object({
   name: z.string(),
 });
 export type CompactEService = z.infer<typeof CompactEService>;
+
+export type AgreementContractPDFPayload = {
```

Comment:
I moved it here from the shared model, because we use it only in the agreement-process and for testing I avoided having to build the models package everytime 😛  In the delegation process we do the same